planet.ini

```diff
@@ -250,4 +250,12 @@ name = Chiranjeev Gupta
 name = Navneet Sharma
 
 [https://medium.com/feed/@abdulwahababrar]
-name = Abdul Wahababrar
\ No newline at end of file
+name = Abdul Wahababrar
+
+[https://medium.com/feed/@shankarj67]
+name = Shankar Jha
+
+
```

Comment:
Only 1 newline is required to mark the end of the file. It was explained the summer training.